docs/config-janitor.rst

```diff
@@ -74,9 +74,11 @@ All configuration options from the general ``[gcp]`` section may be used here.
 
 Additional plugin-specific configuration is needed:
 
-.. option:: zone="STR"
+.. option:: dns_zone="STR"
 
-    `Required`: ID of a Cloud DNS managed zone that instance resource records should belong to.
+    `Required`: DNS zone to validate the correctness of A records before publishing. Must be a fully-qualified domain name (FQDN), ending in ``.``, e.g. ``example.com.``.  If it's a reverse zone, it must be in the form 'A.B.in-addr.arpa.'.  This setting must be either in this section, or in both the ``[gcp.event_consumer]`` and ``[gcp.publisher]`` sections.
```

Comment:
Is "validate the correctness" really the correct wording for how it's used everywhere?  Either way, the section titles are incorrect for janitor.

Reply:
Good catch.

Reply:
Fixed.